docs/docs/rest-api/public/api/v2/types/volumes.raml

```diff
@@ -76,8 +147,8 @@ types:
         type: string
         minLength: 1
         description: The path of the volume in the container
-      external: ExternalVolumeInfo
-      mode:
+      external: ExternalVolumeInfo | CSIVolumeInfo
```

Comment:
I see this is in the `AppExternalVolume`, but in Mesos we support these two types of volumes for both task and task group (pod), I assume Marathon will do the same.

Reply:
Marathon does not have support for external volumes in pods. We will need to add it later.